components/squeezelite/display.c

```diff
@@ -986,6 +998,27 @@ static void visu_update(void) {
 		int level = (visu.bars[0].current + visu.bars[1].current) / 2;
 		draw_VU(display, vu_bitmap, level, 0, visu.row, visu.rotate ? visu.height : visu.width, visu.rotate);		
 	}	
+
```

Comment:
I probably need to better review, but that code will be called all the time, whether there are LEDs or not. We should avoid that then.

Reply:
Done. Added led_vu initialization as condition.